<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: grey;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>200</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border : 2px solid rgba(0,0,0,0);
border-bottom-color :rgba(129,190,190,255);
color:white;
padding-top:16px;



 

</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>310</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border:1px solid black;
border-radius:14px;

}


QPushButton:hover {
	border:1px solid red;
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>591</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="widget_button">
       <property name="text">
        <string>Add new line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>219</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border : 2px solid rgba(0,0,0,0);
border-bottom-color :rgba(129,190,190,255);
color:white;
padding-top:16px;



 

</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>152</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>PMingLiU-ExtB</family>
     </font>
    </property>
    <property name="text">
     <string>Logo drips Icons</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget::item {
	border-bottom:1px solid black;
}

QTableWidget::item:hover {
	border:2px solid red;
}</string>
    </property>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Nouvelle ligne</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Nouvelle colonne</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nouvelle colonne</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nouvelle colonne</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1107</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
